src/com/owncloud/android/ui/fragment/OCFileListFragment.java

```diff
@@ -657,14 +659,70 @@ public int onBrowseUp() {
         return moveCount;
     }
 
+    private void listDirectoryWithAnimationDown(final OCFile file) {
+        Animation fadeOutFront = AnimationUtils.loadAnimation(getContext(), R.anim.dir_fadeout_front);
+        fadeOutFront.setAnimationListener(new Animation.AnimationListener() {
+            @Override
+            public void onAnimationStart(Animation animation) {
+
+            }
+
+            @Override
+            public void onAnimationEnd(Animation animation) {
+                listDirectory(file/*, MainApp.getOnlyOnDevice()*/);
+                Animation fadeInBack =
+                        AnimationUtils.loadAnimation(getContext(), R.anim.dir_fadein_back);
```

Comment:
This line fits properly in the previous one, you do not need to use a line break

Reply:
done